docs/en/token_endpoint.rst

```diff
@@ -302,6 +310,67 @@ The claims available in the *ID Token* are given below.
+    - |spid-icon| |cieid-icon|
+  * - **iat** 
+    - UNIX Timestamp with the time of JWT generation, coded as NumericDate as indicated in :rfc:`7519`.
+    - |spid-icon| |cieid-icon|
+  * - **exp** 
+    - UNIX Timestamp with the expiry time of the JWT, coded as NumericDate as indicated in :rfc:`7519`.
+    - |spid-icon| |cieid-icon|
+  * - **jti** 
+    - It MUST be a String in uuid4 format. Unique *Refresh Token* identifier that the RP MAY use to prevent reuse by rejecting the *Refresh Token*  if already processed.
+    - |spid-icon| |cieid-icon|
+
+.. admonition:: |cieid-icon|
+  
+  **The Refresh Token MUST NOT allow the requesting RP to obtain an ID Token, neither the one previously issued during authentication nor a new ID Token. The use of the Refresh Token MUST NOT be used by RPs to obtain a new user authentication with the OP or to renew a pre-existing session, but MAY be used as a mechanism to obtain from the UserInfo endpoint only the same set of user attributes requested at the initial authentication phase and for which the user has given explicit consent.** Such consent MUST be collected by the OP at the end-user authentication phase on the consent page (it requires the ability to enable or disable this option on the consent page) and MAY be conditional on a validity period if defined by the OP according to policies on the treatment of personal data. Disabling this option by the user MUST still allow user authentication but MUST NOT release a *Refresh Token*. 
+
+  The OP who receives a request for a new *Access Token* via a *Refresh Token* MAY send a notification to the user via one of the available e-delivery services (email, sms, mobile app notification). The user who does not recognize this operation as legitimate or who wants to disable this option can request from the OP a revocation of the given consent (and therefore of the tokens issued as a result of the same) according to the procedures made known within the consent acquisition page. The notification MUST be informative only and not authorizing. Within the notification MUST be made known to the user how to revoke the consent given. The OP MUST allow the user to disable this option at any time through appropriate functionality made available by the OP itself.
+
+For security reasons, an OP MUST return, along with a new *Access Token*, also a new *Refresh Token*, invalidating all previously issued tokens (*Refresh Token Rotation*) to the RP and related to the end-user. The new *Refresh Token* MUST have the *iat* parameter valued with the time instant at which the new request was made and consequently the *exp* parameter based on the expected duration.
+
+Validity Period of a Refresh Token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The *Refresh Token* MUST NOT be valid (difference between *iat* and *exp*) for more than 30 days. 
+
+If at the expiration of the validity period the RP makes a request to the OP, the OP MUST return an error as the result of the request. 
```

Comment:
typo "if at the"

Reply:
Upon expiration of the validity ....

Reply:
done ee6313c3a4eb8491488513df0e61f2d9a86ef669